src/gl/program.c

```diff
@@ -134,7 +134,7 @@ GLuint APIENTRY_GL4ES gl4es_glCreateProgram(void) {
     return program;
 }
 
-void actualy_deleteshader(GLuint shader);
+void actually_deleteshader(GLuint shader);
 void actualy_detachshader(GLuint shader);
```

Comment:
missing the rename on `actually_detachsahder` declaration?

Reply:
good catch, that one has escaped